.gitignore

```diff
@@ -8,3 +8,4 @@ erl_crash.dump
 
 # Downloaded files
 /downloads
+mix.lock
```

Comment:
Asked in the Slack chat and was told it should indeed be kept. So maybe we could skip this bit?